assets/queries/terraform/aws/auto_scaling_group_with_no_associated_elb/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "8e94dced-9bcc-4203-8eb7-7e41202b2505",
+  "queryName": "Auto Scaling Group With No Associated ELB",
+  "severity": "MEDIUM",
+  "category": "Availability",
+  "descriptionText": "AWS Auto Scaling Groups must have associated ELBs to ensure high availability and improve application performance. This means the attribute 'LoadBalancerNames' must be defined and not empty.",
```

Comment:
The attribute should be 'load_balancers', no?

Reply:
Yes, sorry. Done 👍